README.md

```diff
@@ -188,9 +186,207 @@ The procedure is described in `experiments/splash/plot.py`.
 Internally, we use matplotlib, but you're free to use anything else in your experiments - just re-write `plot.py`.
 
 # Example 2: Modifying an existing experiment
+In this example we will modify the splash experiment.
 
-TBD
+In the first part of this example we will:
+ - Change the evaluation from cycles to peak memory consumption.
+ - Change the graph to display absolute numbers.
+
+In the second we will make the benchmarks do a smaller run.
+
+Before starting make sure you have already executed the following:
+```shell script
+$ fex2 template splash
+$ fex2 install splash
+$ cat <<EOF >> build_types/gcc_optimized.mk
+include gcc_native.mk
+
+CFLAGS += -O3
+EOF
+```
+
+## Example 2.1: Changing the evaluation from cycles to peak memory consumption
+To change the evaluation from cycles to peak memory consumption we have to:
+ - Run our SPLASH benchmark with a profiler which can capture peak memory consumption.
+ - Adapt the collection of profile data to our new profile data.
+ - Adapt the graph generation.
```

Comment:
```suggestion
To measure peak memory consumption instead of cycles, we have to:
* Run the benchmarks with a different profiler, the one which can capture peak memory consumption
* Adapt the output parser
* Adapt the plot generation script
```